tests/tests.py

```diff
@@ -44,13 +83,13 @@ def test_replication_and_failover(self):
         # check replication is working by writing rows to the primary
         # and verifying they show up in the replicas
 
-        insert_row = "INSERT INTO tbl1 VALUES ({},'{}');"
+        insert_row = 'INSERT INTO tbl1 (field1, field2) VALUES ({}, "{}");'
```

Comment:
Note to self so I can come back to this: this quoting change was required when I made a `docker exec` call vs Triton. It worked fine vs local Docker, which suggests that there's a subtle divergence in our `docker exec` implementation. I'll investigate this.